src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="stylesheet" href="./styles/style.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__product card__product--image"
+        src="images/imac.jpeg"
+        alt="apple A1419 iMac"
+      >
+      <h1 class="card__product card__product--name">APPLE A1419 iMac 27" Retina <br>5K Monoblock (MNED2UA/A)</h1>
+      <p class="card__product card__product--code">Product code: 195434</p>
+      <section class="card__product card__product--rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <p class="card__product card__product--rating--reviews">Reviews: 5</p>
+      </section>
+      <section class="card__product card__product--price">
+        <p class="card__product card__product--price--title">Price:</p>
+        <p class="card__product card__product--price--number">  $2,199</p>
+      </section>
+      <a
+        href="#"
+        class="card__button"
+        data-qa="hover"
+      >
+        <button class="card__button card__button--buy" type="button">Buy</button>
```

Comment:
Redundant, use only the tag a here
```suggestion
        Buy
```